extensions/wikia/Recirculation/RecirculationHooks.class.php

```diff
@@ -46,7 +46,12 @@ public static function onBeforePageDisplay( OutputPage $out, Skin $skin ) {
 	 * @return bool
 	 */
 	public static function onOasisSkinAssetGroups( &$jsAssets ) {
-		global $wgCityId;
+		global $wgCityId, $wgWikiaEnvironment, $wgNoExternals;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$wgCityId" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)